### DISCUSSION TITLE
Access Certificate

### DISCUSSION THREAD

**Discussion by epiccucumber15** [fastapi/fastapi#6286] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
import uvicorn

from fastapi import FastAPI

app = FastAPI()


@app.get("/")
async def root():
    return {"message": "Hello World"}


if __name__ == '__main__':
    uvicorn.run(
        'server:app', port=443, host='0.0.0.0',
        reload=True,
        ssl_keyfile='code.key',
        ssl_certfile='cert.crt')
```

### Description

How would I access the underlying connection? I want to pin the public certificate so you are unable to access without it.

### Operating System

Windows

### FastAPI Version

0.79.0

### Python Version

Python 3.10.4

**Comment by JarroVGIT** [comment, marked as answer]
This is not a FastAPI question but a Uvicorn question. I would suggest to open an issue in that repository..

**Comment by epiccucumber15** [comment]
ok sorry
